### ISSUE TITLE
unkown error

### ISSUE THREAD

**Issue by venkatgmrit05** [spyder-ide/spyder#7027]
## Description
  File "C:\Users\t3734uk\AppData\Local\Continuum\anaconda3\lib\site-packages\spyder\widgets\mixins.py", line 758, in add_to_history
    encoding.write(text, self.history_filename, mode='ab')
  File "C:\Users\t3734uk\AppData\Local\Continuum\anaconda3\lib\site-packages\spyder\utils\encoding.py", line 228, in write
    textfile.write(text)
**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\Users\t3734uk\AppData\Local\Continuum\anaconda3\lib\site-packages\spyder\widgets\mixins.py", line 758, in add_to_history
    encoding.write(text, self.history_filename, mode='ab')
  File "C:\Users\t3734uk\AppData\Local\Continuum\anaconda3\lib\site-packages\spyder\utils\encoding.py", line 228, in write
    textfile.write(text)
OSError: [Errno 28] No space left on device

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.6.3
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.3 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4

**Comment by ccordoba12** [comment]
> OSError: [Errno 28] No space left on device

The error is clear: you don't have space in your hard drive for Spyder to save your files. So you need to create space for them by deleting something else.

**Comment by CAM-Gerlach** [comment]
Also, duplicate of #6901 which is currently pegged to be fixed at some point.

**Comment by venkatgmrit05** [comment]
Wow! Never thought you guys would reach out personally. Yeah I did clear
the space. Didn't scroll down to end so didn't see the full message before
sending the report.

While I got you here, is there a way to update/install modules to anaconda
while being blocked by office network?

It would help me greatly. Thank you for your time.

Sincerely,
Venkat

**Comment by CAM-Gerlach** [comment]
> Wow! Never thought you guys would reach out personally.

Welcome to Github! Yes indeed, we respond to every single report and do our best to fix it one way or another (or, if we don't have time due to our limited resources, give the users pointers on how to fix/patch it themselves and submit it to be included in Spyder for everyone to benefit). Spyder is a community-supported IDE, developed and funded by users just like you, so we particularly value you helping us discover and track down bugs and other issues, and helping fix your issues in return. I too was but a brand new Spyder user submitting my first issue a mere six months ago, after all.

> While I got you here, is there a way to update/install modules to anaconda while being blocked by office network?

Yes indeed, just get your IT department to unblock it. Its their job to make sure your machine and network functions properly and allows you to do what you need to for your work, so they should be happy to help.

Other than that, this is a bug tracker, and for the Spyder IDE, not Anaconda itself, so you'll be able to get better and more specific advice in a help forum related to that.

Best of luck!

**Comment by venkatgmrit05** [comment]
Alright. Thank you guys. Great work!

Cheers,
Venkat
